lifx_client.py

```
@@ -596,20 +732,75 @@ def set_rgb(self, target: bytes, ip: str, r: float, g: float, b: float,
+                                    time_since_set = time.time() - color_set_time
+                                    if time_since_set >= fade_time - 0.1 and color_set_count <= 2:
+                                        light.state_requested_time = time.time()
+                                        light_to_poll = light
+                            if light_to_poll is not None:
+                                self._request_light_state(light_to_poll)
+                        except Exception as e:
+                            # Log error but don't crash - this is a background thread
+                            import sys
+                            try:
+                                target_str = target.hex() if target else "unknown"
+                            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)